include/ozo/connection.h

```diff
@@ -85,6 +85,7 @@ inline error_code bind_connection_executor(Connection&, const Executor&);
  * @brief Default model for `Connection` concept
  *
  * `Connection` concept model which is used by the library as default model.
+ * The class object is non-copyable.
```

Comment:
This is weak relation with "Remove assign()" :) Please rename commit message

Reply:
**Done**

I've added some extra words to commit message about this.